README.md

```diff
@@ -1 +1,93 @@
+🙏 Acknowledgements
+❓ FAQ (OPTIONAL)
+📝 License
+
+📖 [My mobile version project]
+
+>This project is about creating a portfolio page using best practices for HTML and CSS and uses linters, GitHub Flow rules and documentation of work. It is important to start with a new repository and setup the linters, follow a design template for the portfolio or create your own through Figma and code it using HTML and CSS. I built the mobile website using a Figma template provided by Microverse. I used the linters - GitHub Actions, Lighthouse, Webhint and Stylelint to make this project error-free. Also, all the other files except the HTML and CSS files were added to feature branches to make the main branch remain in best possible condition and stable.
+
+
+## 🛠 Built With
+
+- HTML, CSS
+- No Frameworks
+
+## Tech Stack
+
+- Technologies Used- HTML, CSS, GIT, Linters (GitHub Actions, Lighthouse, Webhint, Stylelint)
+
+## Key Features
+<ol>
+   <li>**Create a new GitHub repository.</li>
+   <li>**Clone the repository to the local env through commandline/terminal or code editors like Visual Studio Code.</li>
+   <li>**Create a new feature branch.</li>
+   <li>**Create .gitignore file.</li>
+   <li>**Set up GitHub Actions and linters in the local env.</li>
+   <li>**Add the main files and folders to the main branch.</li>
+   <li>** Add the following files in separate commits-</li>
+     <ol>
+       <li>A .html file for the skeletal makeup of the webpage.</li>
+       <li>A .css file for styling the webpage.</li>
+     </ol>
+   <li>**Open a pull request.</li>
+   <li>**Submit the project for the code review.</li>
+   <li>**After receiving the review, work on the required changes(if any) and submit again. Keep learning!!!</li>
+</ol>
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Knowledge of HTML, CSS, Linters, Git & GitHub flows and documenting a repo in a professional way
+
+### Setup
+A browser is required to be installed for view the contents and an Code editor to edit the code.
+
+### Install
+You can use this link [https://github.com/azeezumoru/microverse-portfolio.git] to clone my repo
+Clone the repository and use a browser to open the index.html to view the portfolio webpage
+
+👤 **Author1**
+
+- GitHub: [@Alabi12](https://github.com/azeezumoru)
+- Twitter: [@wolo_robert](https://twitter.com/azeez_umoru)
+- LinkedIn: [LinkedIn](linkedin.com/in/umoru-abdul-azeez-527921194)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check this site  https://github.com/azeezumoru/microverse-portfolio.git
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
- [ ] As requested , you should attached a link to the MIT (line 91) to [this file](https://github.com/iambenkis/Portfolio/blob/master/MIT.md) , you can dowload it and customize it for you , but for the NOTE text kindly remove the text to make you README more proffesional :+1: